Gemfile

```diff
@@ -52,7 +52,7 @@ gem "interactor", "~> 3.0"
 gem "sentry-ruby"
 gem "sentry-rails"
 gem "factory_bot_rails"
-gem "active_model_serializers"
+gem "active_model_serializers", github: "vineelvineel/active_model_serializers", branch: "fix_cant_modify_frozen_hash"
```

Comment:
I've merged https://github.com/rails-api/active_model_serializers/pull/2482

Reply:
thank you very much @bf4 :tada: